source/_integrations/spotify.markdown

```diff
@@ -152,3 +153,7 @@ The `media_content_id` value can be obtained from the Spotify desktop app by cli
 ## Unsupported devices
 
 - **Sonos**: Although Sonos is a Spotify Connect device, it is not supported by the official Spotify API.
+
+## Sensors
+
+The Spotify provides sensors that display information of the current song playing.
```

Comment:
```suggestion
Spotify provides sensors that display information about the song that is currently being played. The following sensors are available:

- **Song tempo**: The speed of the piece of music that is currently playing, in beats per minute (bpm).

```
This is the one that I could see. Please correct me if I missed some.

Reply:
I'm going to add more in a followup, will add them and add them here as well